app/js/questions.ts

```diff
@@ -25,6 +27,19 @@ function fillUserTable(HBTemplate: string) {
             fillNav()
             active()
             changeQuestion(current)
+                if (canResumeCookie == '1') {
+                    getAnswersToResume(uid).then(result => {
```

Comment:
this function name is a bit specific. the function actually returns the most recent result for a specific user. That might be useful in other places, but the function name is too specific, instead, name it what it does, not what its used for